### DISCUSSION TITLE
Consider adding README.txt or something for locally cloned wikis

### DISCUSSION THREAD

**Discussion by jcbhmr** [spenserblack/actions-wiki#46] [Ideas]
I just had the random thought to include some kind of a file that would **not be visible on github** but would be if you cloned the wiki locally to say "heyyyy checkout the github repo instead, don't put stuff in the wiki here!"

similar idea i had just now while writing this would be to prepend a `<!-- DO NOT EDIT ON PUBLISHED WIKI, ONLY EDIT IN SOURCE REPO -->\n` or similar comment to all .md files to discourage editing the published version?

these ideas are kinda redundant if the user sees the "github-actions[bot] edited this" thing though... 🤷‍♀️
![image](https://user-images.githubusercontent.com/61068799/227085845-fb429b12-e16e-4a18-b41d-23e43efa6294.png)

**Comment by spenserblack** [comment]
:thinking: Interesting idea. Files that aren't Markdown simply don't get rendered in the wiki, right?
*Maybe* this isn't necessary, since anyone using this action has likely limited wiki edits to collaborators (I'd say this action highly recommends this). For now, maybe wait and see if any other users pop up that need this feature.

However, I initially misinterpreted your idea and had another idea :laughing:
Supporting some sort of way to allow files to be put in the wiki folder, yet get *ignored* when publishing.

**Reply by jcbhmr** [reply]
> 🤔 Interesting idea. Files that aren't Markdown simply don't get rendered in the wiki, right?

yep!

---

sounds like table this idea for now

**Reply by spenserblack** [reply]
Since I think this is valid, but not a priority, #48

**Comment by jcbhmr** [comment]
closing since moved to issue
